ml-agents/mlagents/trainers/sac/trainer.py

```diff
@@ -265,9 +271,8 @@ def update_sac_policy(self) -> None:
             int(self.trainer_parameters["batch_size"] / self.policy.sequence_length), 1
         )
 
-        num_updates = self.trainer_parameters["num_update"]
         batch_update_stats: Dict[str, list] = defaultdict(list)
-        for _ in range(num_updates):
+        while self.step / self.update_steps > self.steps_per_update:
```

Comment:
Is the new hyperparameter enforced as "every time a steps_per_update num of steps elapses, we perform an update" or "after x steps elapses, we perform x/steps_per_update updates."  Judging from this line, is it the latter?

Reply:
Aren't these things the same thing?

Reply:
@andrewcoh @ervteng did you work this out?

Reply:
@ervteng I don't think they are the same thing. If (1) is enforced then (2) is true, but not necessarily is the inverse true. Not sure it's a big deal.